app/services/action_queue.rb

```diff
@@ -146,7 +148,9 @@ def transaction_payload
           },
           order: {
             amount:       data[:amount],
-            currency:     data[:currency]
+            currency:     data[:currency],
+            bank_name:    data[:bank_name],
+            account_number_ending: data[:account_number_ending]
```

Comment:
@paulaferris I totally missed this. You're spot on. I'll shunt them up into the action fields.